README.md

```diff
@@ -29,6 +29,10 @@ npm install replicache
   import Replicache from 'replicache'; // Replace with a real module path as needed...
 
   var rep = new Replicache({
+    // URL of your server endpoint that serves the client view. This is use
+    // by the diff server...
+    clientViewURL: 'https://example.com/replicache-client-view',
```

Comment:
Use https://replicache-sample-todo.now.sh/serve/replicache-batch here instead of example.com.

Reply:
I assume you mean "https://replicache-sample-todo.now.sh/serve/replicache-client-view"

Reply:
oops, yeah